src/varname.jl

```diff
@@ -297,10 +298,18 @@ subsumes(t::PropertyLens, u::PropertyLens) = false
 # FIXME: Does not support `DynamicIndexLens`.
 # FIXME: Does not correctly handle cases such as `subsumes(x, x[:])`
 #        (but neither did old implementation).
-subsumes(t::IndexLens, u::IndexLens) = _issubindex(t.indices, u.indices)
-subsumes(t::ComposedLens{<:IndexLens}, u::ComposedLens{<:IndexLens}) = subsumes_index(t, u)
-subsumes(t::IndexLens, u::ComposedLens{<:IndexLens}) = subsumes_index(t, u)
-subsumes(t::ComposedLens{<:IndexLens}, u::IndexLens) = subsumes_index(t, u)
+subsumes(
+    t::Union{IndexLens,ComposedLens{<:IndexLens}},
+    u::Union{IndexLens,ComposedLens{<:IndexLens}}
+) = subsumes_index(t, u)
+
+
+const subsumedby = (t, u) -> subsumes(u, t)
+const ⊒ = subsumes
+const ⊑ = subsumedby
+const ⋣ = !subsumes
+const ⋢ = !subsumedby
+const ≍ = (t, u) -> t ⋢ u && u ⋢ t
```

Comment:
Are these Unicode operators defined by other packages as well which might cause conflicts? Are they needed?

Reply:
I certainly think in terms of `⊑`, because I don't intuit the argument order of `subsumes`.  And then the rest follows logically.

The question about other packages is a good one; any reasonable way to find out about that besides asking on Slack?

Reply:
BTW I'm not sure about the negated operators. They seem nice to have, but one issue is that they are appearently rendered wrong in some fonts; I see them as `\nsubset`, not `\nsubseteq`, i.e., without the line underneath.

Reply:
Isn't the order the same as `isa`, `issubset`, `in` etc.? Arguably if you read it as infix operator the order of the arguments is quite natural.

Generally, I think Unicode aliases are fine if there are no conflicts with major packages (maybe could be checked with juliahub.com) and if there are ASCII alternatives. Unicode is less accessible (it requires special support for displaying and in particular editing) and therefore in my opinion one should follow the practice in Julia base and always provide an ASCII alternative if some Unicode name is part of the API.

Reply:
Sure, that's why I kept `subsumes` and `subsumedby`.  I'll see whether I can find out about competing definitions.

Reply:
> They seem nice to have, but one issue is that they are appearently rendered wrong in some fonts; I see them as \\nsubset, not \\nsubseteq, i.e., without the line underneath.

I just noticed that they are displayed correctly in the Github app on my phone but incorrectly without the line underneath in Firefox on my linux laptop.

Reply:
Core.Compiler and JET use them for type lattice stuff (pretty much same motivation I applied), and BasicBSpline.jl for (I think) interval containment: https://juliahub.com/ui/Search?q=%E2%8A%91&type=code&f=.%2a%5C.jl